### ISSUE TITLE
Potentially useful library for visual feedback to users

### ISSUE THREAD

**Issue by James-Firth** [jorgearanda/fish#203]
http://ianlunn.github.io/Hover/

Has some nice animations under 'icons' that could be useful to us. Thoughts?

**Comment by nsue** [comment]
Cool. "Icon Down" would look nice on the "Attempt to fish" button.

**Comment by James-Firth** [comment]
Yeah that one and a similar effect for the 'icon float away' would be neat for a successful fishing attempt.

**Comment by jorgearanda** [comment]
Love it—if it's a small effort to bring them in please make it so :-)
